fix: feedbacks from presentation + yseop deploy (#187)
yseop/vscode-yseopml: CHANGELOG.md

## X.X.X (YYYY-MM-DD)

-   Replaced command Yseop Info with Yseop Config.

> -   New command Yseop Libs which installs the libs required by your project.

Reviewer: Currently, there can be at most one, right?

```suggestion
-   New “Yseop Libs” command which installs the YML library required by your project.
```

-   Fix a minor syntax highlighting issue with the `mod` operator.

> -   Replaced snippet `debug` with a new one named `logAttr` which is more explicit and which output is easier to read.

Reviewer: which outputs something
or
whose output is

```suggestion
-   Replaced the `debug` snippet with a new one, named `logAttr`, which is more explicit and whose output is easier to read.
```

> -   Replaced snippet `debug` with a new one named `logAttr` which is more explicit and which output is easier to read.

Reviewer 2: ```suggestion
-   Replaced snippet `debug` with a new one named `logAttr` which is more explicit and whose output is easier to read.
```

## X.X.X (YYYY-MM-DD)

-   Replaced command Yseop Info with Yseop Config.
-   New “Yseop Libs” command which installs the YML library required by your project.

> -   New contextual menu entry on \*.yma file allowing to launch `yseop manager deploy` using Yseop CLI.

Reviewer: ```suggestion
-   New contextual menu entry on `*.yma` files allowing to launch `yseop manager deploy` using Yseop&nbsp;CLI.
```
